src/pymovements/datasets/toy_dataset_eyelink.py

```diff
@@ -115,7 +116,13 @@ class ToyDatasetEyeLink(DatasetDefinition):
         screen_height_cm=30.2,
         distance_cm=68,
         origin='lower left',
-        sampling_rate=1000,
+        eyetracker=EyeTracker(
+            sampling_rate=1000.0,
+            left=True,
```

Comment:
The ToyDataset is only a monocular recording, if I remember it is recorded with the right eye.
Please ask Jakob for confirmation as he did recording.

Here is one example file from the dataset: https://github.com/aeye-lab/pymovements-toy-dataset/blob/main/data/trial_0_1.csv